ddtrace/ddtrace.go

```diff
@@ -79,6 +79,17 @@ type SpanContext interface {
 	ForeachBaggageItem(handler func(k, v string) bool)
 }
 
+// ReadWriteSpan implementations are spans with read/write access.
```

Comment:
"read/write access" to what? Do you just mean spans that are mutable? Let's clarify this.

Reply:
Good point, agreed this could be clearer. 
What I want to convey is that the `ReadWriteSpan` not only lets you write to the span (which is what users are familiar with by using the `ddtrace.Span` interface), but also lets you read from it which isn't possible without this new interface. 
What do you think about this wording:
```
// ReadWriteSpan implementations are spans which can be read from and modified by using the provided methods.
```
?

Reply:
Yeah I think that's a bit clearer 👍

Reply:
Changed in latest commit 👍